lib/samson/hooks.rb

```diff
@@ -25,6 +25,53 @@ def active?
       def require
         super @path
       end
+
+      def add_lib_path
+        engine.config.autoload_paths += Dir["#{engine.config.root}/lib/**/"]
+      end
+
+      def add_decorators
+        engine.config.after_initialize do
+          Dir.glob(engine.config.root.join('app/decorators/**/*_decorator.rb')).each do |c|
```

Comment:
this will load all models and slow down the boot, if we keep it lazy deploy.rb -> calls `add_decorators 'deploy'` -> 'require_dependency' so we can avoid that (should also not need any reload hacks since require_dependency should be reloaded)